src/modules/0.0.5/aws_ec2/entity/instance.ts

```diff
@@ -13,7 +13,6 @@ import {
 import { SecurityGroup, } from '../../aws_security_group/entity';
 import { cloudId, } from '../../../../services/cloud-id'
 
-// TODO complete instance schema
```

Comment:
I have a PR on top of this one where I add more fields, but somehow this change got added on this branch